pkg/cmd/util/clientcmd/factory.go

```diff
@@ -201,7 +201,11 @@ func NewFactory(clientConfig kclientcmd.ClientConfig) *Factory {
 			if !ok {
 				return nil, errors.New("provided options object is not a BuildLogOptions")
 			}
-			buildsForBCSelector := labels.SelectorFromSet(map[string]string{buildapi.DeprecatedBuildConfigLabel: t.Name})
+			buildConfigLabels := labels.Set{
+				buildapi.BuildConfigLabel: t.Name,
```

Comment:
sure, but saving that hop could be a big deal for a big list... saves a marshal/unmarshal step, both of which impact the server (since this is all in-process)